### PR TITLE
[INFINITY-2624] kerberos kafka soak

### PR DESCRIPTION
**Pull request by ArtRand**

### CODE REVIEW

File: tests/test_soak.py [hunk 1]
```diff
@@ -35,6 +41,18 @@ def test_terasort():
         _run_teravalidate()
 
 
+@pytest.mark.soak
+@pytest.mark.rand
```
Comment (susanxhuynh) [inline tests/test_soak.py]: Was this for debugging?

File: tests/test_kafka.py [hunk 2]
```diff
@@ -153,30 +149,45 @@ def kafka_broker_dns():
...
+    return utils.streaming_job_launched(PRODUCER_SERVICE_NAME)
+
+
+def _producer_started():
+    return utils.streaming_job_running(PRODUCER_SERVICE_NAME)
+
+
+def _kafka_broker_dns():
+    cmd = "{package_name} --name={service_name} endpoints broker".format(
+        package_name=KAFKA_PACKAGE_NAME, service_name=KAFKA_SERVICE_NAME)
```
Comment (susanxhuynh) [inline tests/test_kafka.py]: The service name may be different in the soak cluster - "kafka-kerberos". Is it configurable in the test?

File: tests/test_soak.py (hunk 1 above)
Comment (ArtRand) [reply tests/test_soak.py]: whoops.

File: tests/test_kafka.py (hunk 2 above)
Comment (ArtRand) [reply tests/test_kafka.py]: good catch, will change

**Pull request merged**